### DISCUSSION TITLE
How to make assertSnapshot() save to a particular file path

### DISCUSSION THREAD

**Discussion by bwidulski-mdsol** [pointfreeco/swift-snapshot-testing#610] [Q&A]
I'm running a large test that takes screenshots of multiple screens and takes a while to run. I currently use verifySnapshot() to run through it without crashing due to a non-match and assertSnapshot() to make sure screenshots match. However, assertSnapshot() looks in a different folder than the one I save to with verifySnapshot(). How do I make it so that verifySnapshot() saves to a folder and assertSnapshot() looks in that folder.

**Comment by mbrandonw** [comment]
Moving this to a discussion since it is not an issue with the library.

**Comment by jeff77731** [comment]
Same problem

**Comment by alfonsogarza** [comment]
My solution was to mimic what assertSnapshot does but with my directory in verifySnapshot

```
if let failureMessage = verifySnapshot(
    ... your params like
    snapshotDirectory: ... your path,
) { 
  XCTFail(failureMessage)
}
```
